src/lib.rs

```diff
@@ -55,31 +60,14 @@ impl Plugin for WebServerPlugin {
         async_std::task::spawn(run(runtime.clone(), conf));
         Ok(Box::new(WebServerPlugin))
     }
-
-    const STATIC_NAME: &'static str = "webserver";
-}
-impl RunningPluginTrait for WebServerPlugin {
-    fn config_checker(&self) -> zenoh::plugins::ValidationFunction {
-        Arc::new(|name, _, _| {
-            bail!(
-                "Plugin `{}` doesn't support hot configuration changes",
-                name
-            )
-        })
-    }
-    fn adminspace_getter<'a>(
-        &'a self,
-        _selector: &'a Selector<'a>,
-        _plugin_status_key: &str,
-    ) -> ZResult<Vec<zenoh::plugins::Response>> {
-        Ok(Vec::new())
-    }
 }
 
+impl RunningPluginTrait for WebServerPlugin {}
+
 zenoh_plugin_trait::declare_plugin!(WebServerPlugin);
```

Comment:
Should the call to `declare_plugin!()` have a `#[cfg(feature = "no_mangle")]` as in https://github.com/eclipse-zenoh/zenoh-plugin-ros2dds/pull/51 ?
If not I think it's also useless to keep this feature in Cargo.toml

Reply:
Agree. Fixed